packages/react/src/__tests__/server-side-hydrate.test.tsx

```diff
@@ -2,39 +2,52 @@ import React from 'react';
 import { renderToString } from 'react-dom/server';
 import ReactDOM from 'react-dom';
 import { styled } from '@compiled/react';
+import { isNodeEnvironment } from '@compiled/react/dist/runtime/is-node';
+import { useCache } from '@compiled/react/dist/runtime/provider';
+
+jest.mock('@compiled/react/dist/runtime/is-node');
```

Comment:
ahh alternative to `delete process.release.name`. Only thing looks weird is `'@compiled/react/dist/runtime/is-node'` hehe